setup.cfg

```diff
@@ -40,7 +40,7 @@ package_dir =
 
 install_requires =
     importlib-metadata;python_version<"3.8"
-    greenlet != 0.4.17;python_version>='3' and (platform_machine=='arch64' or (platform_machine=='ppc64le' or (platform_machine=='x86_64' or (platform_machine=='amd64' or (platform_machine=='win32' or platform_machine=='WIN32')))))
+    greenlet != 0.4.17;python_version>='3' and (platform_machine=='aarch64' or (platform_machine=='ppc64le' or (platform_machine=='x86_64' or (platform_machine=='amd64' or (platform_machine=='AMD64' or (platform_machine=='win32' or platform_machine=='WIN32'))))))
```

Comment:
we are almost at the point where it may be better to script this -.-'

they could have devised a better way of doing this...